docs/handlers.md

```diff
@@ -54,13 +56,13 @@ Library exports a `State` object that provides a number of constants used to exp
 
 #### `NativeViewGestureHandler` extra properties
 
- - `shouldActivateOnStart`
+ - `shouldActivateOnStart` (**Android only**)
  - `disallowInterruption`
 
 #### `LongPressGestureHandler` extra properties
 
  - `minDurationMs`
- - `maxDist`
+ - `maxDist` (**on Android, may need to set to a very large number, e.g. 5000, to prevent immediate cancellation**)
```

Comment:
is it only Android? 😮

Reply:
As far as I could tell, this only happens on Android. I was not experiencing it on iOS - the default `maxDist` value worked fine.

Reply:
I made a try with maxDist=150 and got expected result. Maybe you have it wrapped with ScrollView?
https://streamable.com/2bj0r